CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Changelog
 Unreleased
 ----------
 ### Changed
-* Update the Firefox test browser to use the `selenium/node-firefox:3.141.59-titanium` docker image
+* Merged global axe rules with default/normal axe rules to allow axe customizations from custom wdio.conf.js
```

Comment:
I'd like to see a note here or in the Wiki that talks about which amongst the two takes precedence - the rules sent as part of a particular spec or the rules configured in the _wdio_ config?

Reply:
@emilyrohrbough, your thoughts on this?

Reply:
anything defined at the test level will be used over the global configuration. I think adding documentation on this behavior would be good. 👍

Reply:
Updated documentation : https://github.com/cerner/terra-toolkit/pull/356/commits/d7d2f237bb503854958d8aa3b9ed8ad76ea95e07